modules/virt-creating-virtualmachineexport.adoc

```diff
@@ -0,0 +1,124 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-export-vms.adoc
+
+:_content-type: PROCEDURE
+[id="virt-creating-virtualmachineexport_{context}"]
+= Creating a VirtualMachineExport custom resource
+
+You can create a `VirtualMachineExport` custom resource (CR) to export the following objects:
+
+* Virtual machine (VM): Exports the persistent volume claims (PVCs) of a specified VM.
+* VM snapshot: Exports PVCs contained in a `VirtualMachineSnapshot` CR.
+* PVC: Exports a PVC. If the PVC is used by another pod, such as the `virt-launcher` pod, the export remains in a `Pending` state until the PVC is no longer in use.
+
+The `VirtualMachineExport` CR creates internal and external links for the exported volumes. Internal links are valid within the cluster. External links can be accessed by using an `Ingress` or `Route`.
+
+The export server supports the following file formats:
+
+* `raw`: Raw disk image file.
+* `gzip`: Compressed disk image file.
+* `dir`: PVC directory and files.
+* `tar.gz`: Compressed PVC file.
+
+.Prerequisites
+
+* The VM must be shut down for a VM export.
+
+.Procedure
+
+. Create a `VirtualMachineExport` manifest to export a volume from a `VirtualMachine`, `VirtualMachineSnapshot`, or `PersistentVolumeClaim` CR according to the following example and save it as `example-export.yaml`:
++
+.`VirtualMachineExport` example
+[source,yaml]
+----
+apiVersion: export.kubevirt.io/v1alpha1
+kind: VirtualMachineExport
+metadata:
+  name: example-export
+spec:
+  source:
+    apiGroup: "kubevirt.io" <1>
+    kind: VirtualMachine <2>
+    name: example-vm
+  ttlDuration: 1h <3>
+----
+<1> Specify the appropriate API group:
++
+* `"kubevirt.io"` for `VirtualMachine`.
+* `"snapshot.kubevirt.io"` for `VirtualMachineSnapshot` or `PersistentVolumeClaim`.
```

Comment:
"" (means empty) for `PersistentVolumeClaim`

Reply:
Kevin Goldblatt requested the change. I'll revert it to empty.